CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required( VERSION 3.1 )
+
+project( ADiKtED LANGUAGES C )
+set(PROJECT_DESCRIPTION "Dungeon Keeper 1 map editor")
+
+include(GNUInstallDirs)
+include(FindPkgConfig)
+
+if( NOT CMAKE_BUILD_TYPE )
+	set( CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE )
+endif()
+
+add_subdirectory(libadikted)
+add_subdirectory(mapslang)
+
+function( add_example ARG_NAME )
+	list( APPEND ARG_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/examples/${ARG_NAME}" )
+	set( SOURCES "" )
+	set( RESOURCES "" )
+	foreach( DIR ${ARG_DIRECTORIES} )
+		file( GLOB GLOB_SOURCES ${DIR}/*.c ${DIR}/*.h )
+		list( APPEND SOURCES ${GLOB_SOURCES} )
+		file( GLOB GLOB_RESOURCES ${DIR}/*.rc )
+		list( APPEND RESOURCES ${GLOB_RESOURCES} )
+	endforeach()
+	add_executable( ${ARG_NAME} ${SOURCES} )
+	target_link_libraries( ${ARG_NAME} PUBLIC
+		adikted ${CMAKE_C_STANDARD_LIBRARIES} )
+	find_package( SDL REQUIRED )
+	target_link_libraries( ${ARG_NAME} PUBLIC ${SDL_LIBRARIES} )
```

Comment:
This needs to be `${SDL_LIBRARY}` or it wont work unless you have CMake 3.19 or above